code/script.js

```diff
@@ -1,9 +1,30 @@
+const accordionItemHeaders = document.querySelectorAll(".accordion-item-header");
+
+accordionItemHeaders.forEach(accordionItemHeader => {
+
+    accordionItemHeader.addEventListener("click", event => {
+        accordionItemHeader.classList.toggle("active"); 
+        const accordionItemBody = accordionItemHeader.nextElementSibling;
+        if(accordionItemHeader.classList.contains("active")) {
+            accordionItemBody.style.maxHeight = accordionItemBody.scrollHeight + "px";
+        }
+        else {
+            accordionItemBody.style.maxHeight = 0;
+        }
+    });
+});
+
```

Comment:
I'm really happy to see the `.forEach` method here.  Imagine if you had 100 accordion items.  This way you wouldn't need 100 copies of the same code, this would be enough! 👍 

Sometimes it's good to add some comments if you don't feel 100% certain of what's going on:

```javascript
// Get all of the accordion items
const accordionItemHeaders = document.querySelectorAll(".accordion-item-header");

// Iterate through each accordion item
accordionItemHeaders.forEach(accordionItemHeader => {

    // Assign an on click listener to each item
    accordionItemHeader.addEventListener("click", event => {
        // Either add or remove the "active" class from the item when clicked
        accordionItemHeader.classList.toggle("active"); 

        // Set the height of the body so the animation activates 
        const accordionItemBody = accordionItemHeader.nextElementSibling;
        if(accordionItemHeader.classList.contains("active")) {
            accordionItemBody.style.maxHeight = accordionItemBody.scrollHeight + "px";
        }
        else {
            accordionItemBody.style.maxHeight = 0;
        }
    });
});
```